### PR TITLE
Pwa support app manifest

### PR DESCRIPTION
**Pull request by maier49**
**Type:** feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [x] Unit or Functional tests are included in the PR

**Description:**

Partially addresses dojo/meta#170, the manifest and images also need to be copied into the build project to fully satisfy it. [This page](https://developers.google.com/web/fundamentals/engage-and-retain/app-install-banners/) describes what is necessary in an app manifest for it to prompt the user to install the application, but the lighthouse test itself asserts slightly different and additional criteria. Where the two differed I've tried to satisfy the lighthouse checks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,16 @@ For testing the application run `dojo test`, which will build the application (`
...
 
+### Manifest.json
+
+This sample app contains [a web app manifest](https://developer.mozilla.org/en-US/docs/Web/Manifest), `manifest.json`, that allows the app
+to be installed on the homescreen of a mobile device if it was built with Service Workers enabled. Service Workers can be enabled by setting the `"serviceWorker"` option in the `.dojorc` config for `build-webpack`.
+
+By default, the manifest will use the images in the images folder as the app's icon if it is installed. These images can and should be replaced
+with an appropriate icon for your application.
+
+It also has a `background_color` and `theme_color` in `manifest.json`, which can be updated to match the desired color scheme for your application.
```
Comment (dylans) [inline README.md]: It also -> The sample app also

**Review by dylans — CHANGES_REQUESTED** [review]
one small readme nit

**Comment by maier49** [comment]
@dylans If you can take another look I've fixed the issue you pointed out.

**Comment by kitsonk** [comment]
Yeah, like the other one... I am wondering if we have an _offline_ or _PWA_ template.  The other thing we should consider is what logo to use by default.  Since this is downstream users and not ourselves, we should have Scott/Torrey come up with some sort of generic app icon that doesn't actually make it potentially feel like it is an official "Dojo" thing when someone publishes it.

**Comment by maier49** [comment]
@kitsonk The files and changes included here don't cost the developer anything if they don't use them, but they leave open the option of opting in to service worker usage later. By default `registerServiceWorker` is a noop, and the manifest link is ignored in at least chrome, Firefox and IE 11. In my opinion including these is akin to including empty test files and test config. We're not forcing you to use them but they're there if you decide to take advantage.

**Comment by maier49** [comment]
In light of the concerns mentioned in [this comment](https://github.com/dojo/meta/issues/83#issuecomment-321250186), we will take a different approach for supporting service workers and other PWA features. It will involve using `.dojorc` and generating appropriate files in `cli-build-webpack`, as opposed to adding code to `cli-create-app`  that may not apply in all cases.

**Pull request closed without merging**